rockspec/apisix-master-0.rockspec

```diff
@@ -59,6 +59,7 @@ dependencies = {
     "lua-resty-expr = 1.0.0",
     "graphql = 0.0.2",
     "argparse = 0.7.1-1",
+    "luasocket = 3.0rc1-2",
```

Comment:
I think we should use a `table` version

Reply:
This pkg was imported in #2965, maybe we could fix it in another pr